### PR TITLE
ENH: Update extension metadata

### PR DESCRIPTION
**Pull request by jcfr**
Consolidate extension metadata based on the corresponding s4ext file organized
in the ExtensionsIndex repository.

### CODE REVIEW

**Comment by jcfr** [comment]
Please, could this be reviewed & merged :pray:

**Comment by jcfr** [comment]
Email sent on June 13th 2024 to `posgrad@ortodontia.ufrj.br` with @lassoan and @bpaniagua in cc:

```
Subject: Help to maintain the Slicer extension "SlicerOrthodonticAnalysis"

Dear Ortodontia staff, 

I am trying to get in touch with any of the following individuals:

- Joao Vitor Lima Coimbra (Federal University of Espirito Santo)
- Vinicius Campos de Oliveira Batista (Federal University of Espirito Santo)
- Rafhael Milanezi de Andrade (Federal University of Espirito Santo)
- Pedro Lima Emmerich Oliveira (Federal University of Rio de Janeiro)
- Lincoln Issamu Nojima (Federal University of Rio de Janeiro)

who are referenced as being contributors to the [SlicerOrthodonticAnalysis](https://github.com/OrthodonticAnalysis/SlicerOrthodonticAnalysis/blob/20ade92d1d083da5587a1d3529d89596ff29641b/CMakeLists.txt#L9) extension.

Question: Could you forward this email to at least one of these individuals ? For example Pedro Lima Emmerich Oliveira, who seems to be working in your department. See https://scholar.google.com.br/citations?user=2E7N3tIAAAAJ&hl=pt-BR

For reference, I found your email looking at the archive of the following website:
https://web.archive.org/web/20191220170205/http://www.ortodontia.odontologia.ufrj.br/pos-graduacao

To summarize, we would appreciate your help to merge the following pull request:

  ENH: Update extension metadata
  https://github.com/OrthodonticAnalysis/SlicerOrthodonticAnalysis/pull/4

Changes are trivial as they only include updates to the extension metadata.

Thanks for the help,
J-Christophe
```

**Review by pedroemmerich — APPROVED** [review]

**Review by pedroemmerich — APPROVED** [review]

**Comment by jcfr** [comment]
Thanks for approving 🙏, next it would be great if could merge the changes 🚀

**Comment by jamesobutler** [comment]
@pedroemmerich Would you be able to complete the integration of this PR? Thanks!
